backend/src/main/java/hu/flowacademy/konyhatunder/service/RecipeService.java

```diff
@@ -115,6 +114,27 @@ private String translateUnit(Measurement measurement, String unit) {
         return difficulties;
     }
 
+    public List<Recipe> sendRecipesByIngredients(List<Ingredient> ingredientList) {
+        validateReceivedIngredients(ingredientList);
+        List<Recipe> foundRecipes = new ArrayList<>();
```

Comment:
Ha ez egy set, akkor nem kell bűvészkedni alant, hanem csak beleömleszteni, majd listává alakítani. Érdemes majd egy rendezést is megadni.

Reply:
Done.